meinberlin/apps/projects/signals.py

```diff
@@ -0,0 +1,49 @@
+from django.core.cache import cache
+from django.db.models.signals import post_save
+from django.db.models.signals import pre_delete
+from django.dispatch import receiver
+
+from adhocracy4.dashboard import signals as a4dashboard_signals
+from adhocracy4.projects.models import Project
+from meinberlin.apps.projects.tasks import get_next_project_ends
+from meinberlin.apps.projects.tasks import get_next_projects_start
+
+
+@receiver(a4dashboard_signals.project_created)
+@receiver(a4dashboard_signals.project_published)
+@receiver(a4dashboard_signals.project_unpublished)
+def post_dashboard_signal_delete(sender, project, user, **kwargs):
+    cache.delete_many(
+        [
+            "projects_active*",
```

Comment:
do we need the globbing here? the wildcard will only match `projects_activeParticipation`, right ? And the wildcard will make it incompatible with non-redis cache backends